src/__tests__/ReduxRouter-test.js

```diff
@@ -223,8 +223,6 @@ describe('<ReduxRouter>', () => {
       'or a class/function (for composite components) but got: object. ' +
       'Check the render method of `ReduxRouterContext`.';
 
-    const routingContextInvalidPropErrorMessage = 'Invalid prop `RoutingContext` of type `object` supplied to `ReduxRouterContext`';
```

Comment:
@halt-hammerzeit I had to remove this error message after the babel6 update. Could you take look?